openslides_backend/permissions/permissions.py

```diff
@@ -11,6 +11,26 @@ class OrganisationManagementLevel(str, Enum):
     CAN_MANAGE_USERS = "can_manage_users"
     CAN_MANAGE_ORGANISATION = "can_manage_organisation"
 
+    @classmethod
+    def get_level_number(
+        cls, oml: "OrganisationManagementLevel", default_level: int = 0
+    ) -> int:
+        return OrganisationManagementLevel_numbers.get(oml, default_level)
+
+
+OrganisationManagementLevel_numbers = {
+    OrganisationManagementLevel.SUPERADMIN: 3,
+    OrganisationManagementLevel.CAN_MANAGE_ORGANISATION: 2,
+    OrganisationManagementLevel.CAN_MANAGE_USERS: 1,
+}
+
+
+class CommitteeManager:
+    """ 2nd Permission Type, implemented as User.committee_as_manager_ids """
+
+    def __str__(self) -> str:
+        return "Committee Manager Right"
+
```

Comment:
2 remarks:
1. This is a calculated file, just like `models.py`. Please add everything here to `generate-permissions.py`.
2. I would suggest to define `CommitteeManager` as an enum analogous to `OrganisationManagementLevel` for consistency. The enum will only hold one member for now but that's ok. So something like `CommitteeManagementLevel` with the member `Manager = 1`.

Reply:
(the latter one of course has to be also added to the generation file)

Reply:
Oh yes, that's really bad

Reply:
Done